tests/Transport_Steady/Transport2D_5PolyA_AniHeteroBndry.lua

```diff
@@ -0,0 +1,190 @@
+-- 2D Transport test with Vacuum and Incident-isotropic BC.
```

Comment:
thanks for adding a test! I can't wait to use this soon!

Reply:
:-)